### PR TITLE
Docs invoice default expiry

### PR DESCRIPTION
**Pull request by YusukeShimizu**
## Change Description
https://github.com/lightningnetwork/lnd/issues/7278

## Steps to Test
N/A

### Testing
~~- [ ] Your PR passes all CI checks.~~
~~- [ ] Tests covering the positive and negative (error paths) are included.~~
~~- [ ] Bug fixes contain tests triggering the bug to prevent regressions.~~

### Code Style and Documentation
- [X] The change obeys the [Code Documentation and Commenting](https://github.com/lightningnetwork/lnd/blob/master/docs/code_contribution_guidelines.md#CodeDocumentation) guidelines, and lines wrap at 80.
- [X] Commits follow the [Ideal Git Commit Structure](https://github.com/lightningnetwork/lnd/blob/master/docs/code_contribution_guidelines.md#IdealGitCommitStructure).
~~- [ ] Any new logging statements use an appropriate subsystem and logging level.~~
- [X]  [There is a change description in the release notes](https://github.com/lightningnetwork/lnd/tree/master/docs/release-notes), or `[skip ci]` in the commit message for small changes.

📝 Please see our  [Contribution Guidelines](https://github.com/lightningnetwork/lnd/blob/master/docs/code_contribution_guidelines.md) for further guidance.

### CODE REVIEW

**Review by ellemouton — CHANGES_REQUESTED** [review]
Thanks for the PR @YusukeShimizu! Looking good!
Please see our [contributor guidelines](https://github.com/lightningnetwork/lnd/blob/master/docs/code_contribution_guidelines.md#model-git-commit-messages) for how to title commits. 

For example, I think the first commit could be titled something like: `lncli: fix addinvoice expiry flag comment` and the second one could be `lnrpc: update invoice request expiry comment` or something along those lines. 

Since this PR touches generated GRPC code, I would say that we should not skip the CI. 

It is also missing an entry in the release notes

**Comment by YusukeShimizu** [comment]
@ellemouton 
I fixed commit messages and added release note.  

Some CIs are failed.  
I believe retry is the solution.  
Can you please tell me how to do this?

File: cmd/lncli/cmd_invoice.go [hunk 1]
```diff
@@ -57,7 +57,8 @@ var addInvoiceCommand = cli.Command{
 		cli.Int64Flag{
 			Name: "expiry",
 			Usage: "the invoice's expiry time in seconds. If not " +
-				"specified an expiry of 3600 seconds (1 hour) " +
+				"specified, an expiry of " +
```
Comment (ellemouton) [inline cmd/lncli/cmd_invoice.go:60]: nit: could probably fit this string on just 2 lines by not wrapping so early. same for below

File: lnrpc/invoicesrpc/invoices.swagger.json [hunk 2]
```diff
@@ -261,7 +261,7 @@
         "expiry": {
           "type": "string",
           "format": "int64",
-          "description": "Payment request expiry time in seconds. Default is 3600 (1 hour)."
+          "description": "Payment request expiry time in seconds. Default is 86400 (24 hour)."
```
Comment (ellemouton) [inline lnrpc/invoicesrpc/invoices.swagger.json:264]: nit: s/hour/hours (same for below)

**Review by ellemouton — COMMENTED** [review]
Looking good! Left a few nits. 
Dont need to worry about the CI fails - they are known flakes.

**Review by yyforyongyu — COMMENTED** [review]
Thanks for the PR! This should be good to go once @ellemouton 's nits are fixed!

File: cmd/lncli/cmd_invoice.go (hunk 1 above)
Comment (YusukeShimizu) [reply cmd/lncli/cmd_invoice.go:60]: I understood that your suggestion is following
```suggestion
				"specified, an expiry of 86400 seconds (24 hours) is implied."
```

By this change,`line is 95 characters (lll)` is detected by go-golangci-lint.  
Could you show me any suggestion?

Comment (ellemouton) [reply cmd/lncli/cmd_invoice.go:60]: looking better now! was originally spread across more lines.

**Review by ellemouton — APPROVED** [review]

**Pull request merged**